libr/debug/debug.c

```diff
@@ -1359,7 +1359,8 @@ R_API int r_debug_child_clone(RDebug *dbg) {
 }
 
 R_API bool r_debug_is_dead(RDebug *dbg) {
-	bool is_dead = (dbg->pid == -1 && strncmp (dbg->h->name, "gdb", 3));
+	bool is_dead = (dbg->pid == -1 && strncmp (dbg->h->name, "gdb", 3))
+		|| (dbg->reason.type == R_DEBUG_REASON_DEAD);
```

Comment:
ping @skuater do you know why ? It sounds logic to me that r_debug_is_dead shall return true if dbg->reason.type is R_DEBUG_REASON_DEAD, but it actually totally breaks the debugger.